assets/css/app.css

```diff
@@ -7,8 +7,10 @@ html, body, #content-wrapper, .page-content, #view {
   white-space: normal !important;
 }
 
-.btn-group button {
-  margin: 3px;
+.btn-group button,
```

Comment:
I believe these changes are not required anymore.

Reply:
Since the buttons are located in the header, removing top and bottom margins is required in order to avoid the header becoming higher.

Reply:
Since the buttons are located in the header, removing top and bottom margins is required in order to avoid the header becoming higher.

Reply:
As we're not using `<buttons>` anymore but labels (see my comment above), these changes are not required anymore.